### ISSUE TITLE
upgrading to @swc/helpers v0.3.0

### ISSUE THREAD

**Issue by doums** [swc-project/swc#2986]
### Describe the bug

Hi,

This is not directly related to swc. I just tried to upgrade `@swc/helpers` to the new minor version `0.3.0`. But after this `ts-node`, which we use to run our apps in dev environment, stop working with the following error

```shell
> ts-node --transpile-only -r dotenv/config ./src/index.ts --nexus-generate

/home/pierre/Documents/aos/files/node_modules/@swc/helpers/lib/index.js:1
export { default as applyDecoratedDescriptor } from './_apply_decorated_descriptor';
^^^^^^

SyntaxError: Unexpected token 'export'
    at Object.compileFunction (node:vm:352:18)
    at wrapSafe (node:internal/modules/cjs/loader:1031:15)
    at Module._compile (node:internal/modules/cjs/loader:1065:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Object.require.extensions.<computed> [as .js] (/home/pierre/Documents/aos/files/node_modules/ts-node/src/index.ts:1361:43
)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
```
With the previous version, there is no problem.

This is my tsconfig that is used by ts-node:
```
{
  "compilerOptions": {
    "esModuleInterop": true,
    "sourceMap": true,
    "strict": true,
    "module": "commonjs",
    "moduleResolution": "node",
    "resolveJsonModule": true,
    "target": "ES2020"
  }
}
```

### Config

```json
{
  "jsc": {
    "parser": {
      "syntax": "typescript",
      "dynamicImport": true
    },
    "target": "es2020",
    "externalHelpers": true
  },
  "module": {
    "type": "commonjs"
  },
  "minify": true,
  "sourceMaps": true
}
```

### Expected behavior

Upgrading to the last version of @swc/helpers should not prevent ts-node to run my code.

### Version

last

**Comment by kdy1** [comment]
Can you try `v0.3.1`?

**Comment by doums** [comment]
```
Error: Cannot find module './_apply_decorated_descriptor'
Require stack:
- /home/pierre/Documents/aos/files/node_modules/@swc/helpers/lib/index.js
- /home/pierre/Documents/aos/files/node_modules/@go-aos/logger/dist/index.js
- /home/pierre/Documents/aos/files/services/image-service-gql/src/index.ts
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._resolveFilename.sharedData.moduleResolveFilenameHook.installedValue [as _resolveFilename] (/home/pierre/Documents/aos/f
iles/node_modules/@cspotcode/source-map-support/source-map-support.js:679:30)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/home/pierre/Documents/aos/files/node_modules/@swc/helpers/lib/index.js:413:57)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Object.require.extensions.<computed> [as .js] (/home/pierre/Documents/aos/files/node_modules/ts-node/src/index.ts:1361:43)
    at Module.load (node:internal/modules/cjs/loader:981:32) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/pierre/Documents/aos/files/node_modules/@swc/helpers/lib/index.js',
    '/home/pierre/Documents/aos/files/node_modules/@go-aos/logger/dist/index.js',
    '/home/pierre/Documents/aos/files/services/image-service-gql/src/index.ts'
  ]
}

```

**Comment by kdy1** [comment]
Will be fixed by `v0.3.2`

**Comment by doums** [comment]
Ok I confirm this is fixed now, thanks !
